srcpkgs/calamares/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'calamares'
+pkgname=calamares
+version=3.2.44.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release
+                -DCMAKE_INSTALL_PREFIX=/usr
```

Comment:
Do intend with only 1 space inside a variable.

Reply:
I am not entirely sure what you mean by that?

Reply:
Don't align to quote with many spaces, use just 1.